### DISCUSSION TITLE
How to get data from bff/v2/mobile-app endpoint?

### DISCUSSION THREAD

**Discussion by ehendrix23** [tdorssers/TeslaPy#65] [Q&A]
How can one use the bff/v2/mobile-app endpoints such as bff/v2/mobile-app/charging/history/charging/history?

I tried both https://akamai-apigateway-vfx.tesla.com/bff/v2/mobile-app/charging/history and https://owner-api.teslamotors.com/bff/v2/mobile-app/charging/history but I keep on getting HTTP client error 404.

As per #61, I did change the header in __init__.py to TeslaApp/4.7.0 for x-tesla-user-agent.

Anyone can provide working example?

**Comment by tdorssers** [comment]
I will make x-tesla-user-agent customizable.

**Comment by ehendrix23** [comment]
But even when updating the header myself I still did not get it to work so assuming I'm doing something wrong. Any examples available?
